server/utils/bedUtils.ts

```diff
@@ -21,23 +22,27 @@ export const bedsActions = (premisesId: string, user: UserDetails) =>
       ]
     : null
 
-export const bedNameCell = (item: Cas1PremisesBedSummary): TableCell => ({ text: item.bedName })
-
-export const roomNameCell = (item: Cas1PremisesBedSummary): TableCell => ({ text: item.roomName })
+export const bedsTableRows = (beds: Array<Cas1PremisesBedSummary>, premisesId: string) => {
+  const sorted = beds.sort(({ bedName: name1 }, { bedName: name2 }) =>
+    name1.localeCompare(name2, 'en', { numeric: true, sensitivity: 'base' }),
+  )
 
-export const actionCell = (bed: Cas1PremisesBedSummary, premisesId: string): TableCell => ({
-  html: bedLink(bed, premisesId),
-})
+  return sorted.map(bed => [
+    htmlCell(bedLink(bed, premisesId)),
+    textCell(bed.roomName),
+    htmlCell(characteristicsBulletList(bed.characteristics, { classes: 'govuk-list--compact' })),
+  ])
+}
```

Comment:
Much nicer than those ultra-specific `bedNameCell` and `roomNameCell` classes. I've been sheepish about removing similar utilities elsewhere which I didn't find very useful; I should probably have gone ahead and got rid of them!

I'd even say we could probably get rid of `bedLink` -- if a utility is only calling another, existing one, without doing any more work, it's possibly unnecessary?